<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OlinScheduler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Code :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="code"/>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Title :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="title"/>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Instructors :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="instructors"/>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Credit :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="credit">
      <property name="toolTip">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_10">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>My Courses : </string>
       </property>
      </widget>
      <widget class="QListWidget" name="myCourses"/>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="removeBtn">
      <property name="toolTip">
       <string>Remove selected course from my list</string>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearBtn">
      <property name="toolTip">
       <string>Clear all of my courses</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="colorBtn">
      <property name="toolTip">
       <string>Apply color to my courses</string>
      </property>
      <property name="text">
       <string>Colorize</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="openSchedule">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>246</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Opens Schedule Window.</string>
    </property>
    <property name="text">
     <string>Schedule View</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QComboBox" name="pool">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The List of All Available Courses</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Add selected course to my courses</string>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Create a new course entry</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBtn">
     <property name="toolTip">
      <string>Delete selected course entry</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_Save">
     <property name="title">
      <string>&amp;Save</string>
     </property>
     <addaction name="actionCourses"/>
     <addaction name="actionSchedule"/>
    </widget>
    <addaction name="action_Open"/>
    <addaction name="menu_Save"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionTitle"/>
    <addaction name="actionCode"/>
    <addaction name="actionInstructor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
  </action>
  <action name="actionTitle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </action>
  <action name="actionInstructor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Instructor</string>
   </property>
  </action>
  <action name="actionSchedule">
   <property name="text">
    <string>Schedule</string>
   </property>
   <property name="toolTip">
    <string>Save The Image of the Schedule as a .png file</string>
   </property>
   <property name="statusTip">
    <string>Save The Image of the Schedule as a .png file</string>
   </property>
  </action>
  <action name="actionCourses">
   <property name="text">
    <string>Courses</string>
   </property>
   <property name="toolTip">
    <string>Saves List of All the Courses currently in the pool.</string>
   </property>
   <property name="statusTip">
    <string>Saves List of All the Courses currently in the pool.</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="statusTip">
    <string>Loads a list of courses from a file.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
